dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/Constants.java

```diff
@@ -699,8 +699,10 @@ private Constants() {
     /**
      * application regex
      */
-    public static final String APPLICATION_REGEX = "application_\\d+_\\d+";
+    /* public static final String APPLICATION_REGEX = "application_\\d+_\\d+"; */
+    public static final String APPLICATION_REGEX = "http[s]?://([\\w-]+\\.*)+[\\w-]+(/[\\w-.*/?%&=]*)?:(\\d*)/(\\w*)/application[_]?\\d+[_]?\\d+";
```

Comment:
I think the regular expression cannot be modified in this way. The yarn application id matching will involve tasks submitted by calculation engines such as flink, hive, spark, etc., and the printed yarn application id log formats are different. The following is the yarn application id printed by flink submitted tasks. , The modified regularity cannot be matched
```
INFO  org.apache.flink.yarn.YarnClusterDescriptor                   - Found Web Interface xxxx of application 'application_1625147587008_396985'.

```